docs/pix2struct-chyrons/v1.0/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Pix2struct Chyrons",
+  "description": "extracts text from input timeframes based on chyron queries using the pix2struct Doc-VQA model",
+  "app_version": "v1.0",
+  "mmif_version": "1.0.0",
+  "analyzer_version": "1",
```

Comment:
Just to ensure the replicability, we generally want to use a permanently versioned model. Namely, I think we should amend [this line](https://github.com/clamsproject/app-pix2struct-chyrons/blob/cd959fb8b7954769de590924670cdb53e9c475c0/app.py#L20) using [this arg](https://huggingface.co/docs/transformers/v4.32.0/en/main_classes/model#transformers.PreTrainedModel.from_pretrained.revision) with a specific model version from [here](https://huggingface.co/google/pix2struct-docvqa-base/commits/main), and use the same version number here in `analyzer_version` value.